terminologies/technique/digitalReconstructionTechnique.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/digitalReconstructionTechnique",

```

Comment:
this one would be the only I would register for all 3D and 2D reconstruction techniques. I would change it maybe to "digitalVolumeReconstruction" (vs. "digitalVectorReconstruction" below)

@tgbugs: what do you think?

Reply:
Changed to "reconstruction technique" in PanProject list.